test/test_broker/mocks/cloudController.js

```diff
@@ -325,6 +325,7 @@ function findServicePlanByInstanceId(instance_id, plan_guid, plan_unique_id, res
     .query({
       q: `service_instance_guid:${instance_id}`
     })
+    .times(times)
```

Comment:
do it like this instead `.times(times || 1)`

Reply:
okay .. will update.